x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch_settings/check/internal_monitoring.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CatIndicesParams } from 'elasticsearch';
+// @ts-ignore
+import { handleError } from '../../../../../lib/errors';
+import { RouteDependencies } from '../../../../../types';
+
+export function internalMonitoringCheckRoute(_server: unknown, npRoute: RouteDependencies) {
+  npRoute.router.get(
+    {
+      path: '/api/monitoring/v1/elasticsearch_settings/check/internal_monitoring',
+      options: { tags: ['access:monitoring'] },
```

Comment:
We should remove this. I did for the other alert apis too: https://github.com/elastic/kibana/pull/71942/commits/ba48a4a82f78e181d636e9d58359382166d324d3